src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesLineObsolete.TableExt.al

```diff
@@ -0,0 +1,61 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+/// <summary>
+/// This object contains the obsoleted elements of the "No. Series" table.
+/// </summary>
+tableextension 309 NoSeriesLineObsolete extends "No. Series Line"
+{
+    fields
+    {
+    }
+
+#if not CLEAN24
```

Comment:
Shouldn't the CLEAN symbols encapsulate the entire file here? From what I see, it will just be an empty table extension after the code between the clean symbols is removed.

Reply:
We are planning on moving some fields here, may happen in the next iteration though.